listenstore/listenstore/listenstore.py

```diff
@@ -31,12 +31,10 @@ def __init__(self, conf):
     def max_id(self):
         return int(self.MAX_FUTURE_SECONDS + calendar.timegm(time.gmtime()))
 
-
-    def fetch_listens_from_storage():
+    def fetch_listens_from_storage(*args):
```

Comment:
The overridden method had arguments but this base method did not. I added `*args` to denote that this method accepts multiple arguments.